modules/ROOT/pages/exception-filter.adoc

```diff
@@ -4,8 +4,55 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: error handling, exceptions, exception catching, exceptions, filter
 
-*Purpose:* Filters messages according to the type of an exception.
+The Exception Filter allows filtering of messages according to the type of an exception.
 
-*Minimum Configuration:* The class that indicates the type of the exception.
+Prerequisites: You need a Catch Exception Strategy configured in your flow.
 
-*Discussion:* An Exception Filter matches on the type of an exception. You supply the class indicating the exception type to the Expected Type property. For example, for a runtime exception type, you might set Expected Type to `java.lang.RuntimeException`. You can enter the class name or browse to it.
\ No newline at end of file
+Minimum Configuration: The class that indicates the type of the exception.
+
+An Exception Filter matches the type of exception you configure in the *Expected Type* field in the *Global Element* properties in Studio, or in the `expectedType` property in the Configuration XML.
+
+The Exception Filter expects an exception; this means that you need to add it within a Catch Exception Strategy processor. The exception is passed to the next processor in the exception handler only if it matches the Exception Filter's configured `expectedType`.
+
+Use the Exception Filter to decide if your Catch Exception Strategy needs to handle an exception or not, based on the type of the exception. When the Exception Filter receives an exception that does not match its configured `expectedType`, it returns a `null` value, and the processing of that message stops.
+
+== Configure the Filter
+
+. In Anypoint Studio, search for "filter" in the Palette and drag the Filter Reference block to the Catch Exception Strategy:
++
+image::exception-filter-flow.png[]
++
+. Click the Filter Reference processor to open the properties editor:
++
+image::exception-filter-properties.png[]
++
+. Click the green plus sign to the right of *Global Reference*.
+. Click the arrow next to *Filters* to show the filter selections:
++
+image::filter-global-type.png[]
+. Click *Exception* and *OK*.
+. Configure the class that indicates the type of the exception in the *Expected Type* field:
++
+image::exception-filter-global.png[]
++
+[TIP]
+For example, for a runtime exception type, you set *Expected Type* to `java.lang.RuntimeException`. You can enter the class name or browse to it.
+
+== Example Code
```

Comment:
Maybe "Example Code" -> "XML Example"

same for "Code sample" to "XML example" or similar